src/Neo/SmartContract/StorageKey.cs

```diff
@@ -311,13 +325,13 @@ public override int GetHashCode()
 
         public byte[] ToArray()
         {
-            if (_cache is null)
+            if (_cache.IsEmpty)
```

Comment:
```suggestion
            if (_cache is { IsEmpty: true })
```

`ReadOnlyMemory<byte> bad = MemoryMarshal.AsMemory(null);`